Tile.cpp

```diff
@@ -186,18 +104,6 @@ bool Tile::IsValid()
     return valid;
 }
 
-/**
```

Comment:
This function was originally used for marking tiles as unused; it was removed because it was almost guaranteed to leak memory (since it would mark the objects as freed without calling the destructor).

I suppose in theory this could've been used instead of manually setting `valid=false;` on every tile immediately after allocation, but it would've been significantly slower.